README.md

```diff
@@ -201,13 +201,25 @@ inputs:
         name_format: "{var_name}"
     target_output_variable: static
 
+extra:
+  projection:
+    class_name: LambertConformal
+    kwargs:
+      central_longitude: 25.0
```

Comment:
It might be that we are actually still missing some info here, because from the GRIB files I can see that there is a `false_easting` and `false_northing` different to 0 (the default).

```suggestion
      central_longitude: 25.0
      false_easting: 1999248.19322605
      false_northing: 609541.209132949
```
I have opened an issue with frie data as they also ignore this info in their definition for DINI. @SimonKamuk would be great to have your expertise here.

Reply:
Ah! I didn't realise we are missing this too.... oh boy. Yes, @SimonKamuk, would be great to know what you think :)

Reply:
![image](https://github.com/user-attachments/assets/c4d84436-1722-4dcb-87b2-19d1a641949e)
if I uncomment false_easting and false_northing, the third and fourth assertions fail. So to me it seems either I am doing something wrong or the grib messages are wrong - neither would surprise me 😆

Reply:
![wkt_obj_nofalsing_customGlobe](https://github.com/user-attachments/assets/b68154bd-017d-48dd-a7dc-27188062cb66)
![wkt_obj_wfalsing_customGlobe](https://github.com/user-attachments/assets/d59906f4-9bd2-4696-b001-d1b1bfd25e18)
I went the visual way now and the one that fits, is the one with the northing and easting set to 0.

```
import pyproj
import cartopy
import matplotlib.pyplot as plt
PyProj =  pyproj.crs.CRS.from_wkt(wkt)
CartopyProj = cartopy.crs.Projection(PyProj)
fig, ax = plt.subplots(subplot_kw=dict(projection=ccrs.PlateCarree()))
ds.t0m.isel(time=10).plot(ax=ax, x="x", y="y", transform=CartopyProj)
ax.coastlines()
ax.gridlines(draw_labels=["top", "left"])
fig.show()
```

Reply:
I am more and more convinced that the grid definition in the GRIB files of DANRA is incorrect, as the smallest x value is -1999248.193226 and the smallest y value is -609541.209133, exactly the numbers of the false_northing and false_easting. Because these numbers are kind of old school and [have been used to avoid negative numbers](https://help.arcgis.com/en/geodatabase/10.0/sdk/arcsde/concepts/geometry/coordref/coordsys/projected/mapprojections.htm), the `false_easting` and `false_northing` should have been subtracted, which obviously did not happen here.

Reply:
@leifdenby @SimonKamuk thanks for the additional discussion. I feel better now about this definition reflecting the reality to the best of our knowledge 🤣

Reply:
Also, I wonder who we need to inform about these incorrect GRIB definitions? I supposed they will be wrong when we share the GRIB files too... but that is something to work out somewhere else 😄